tests/client_test.go

```diff
@@ -32,14 +32,21 @@ func TestStreamClient(t *testing.T) {
 	streamServer := client.StreamUpstreamServer{
 		Server: "127.0.0.1:8001",
 	}
-	// test adding a stream server
-
-	err = c.AddStreamServer(streamUpstream, streamServer)
 
+	streamServers, err := c.GetStreamServers(streamUpstream)
 	if err != nil {
-		t.Fatalf("Error when adding a server: %v", err)
+		t.Errorf("Error getting stream servers: %v", err)
 	}
 
+	// test adding a stream server
+
+	if len(streamServers) < 1 {
```

Comment:
hm why do we ned this block `if len(streamServers) < 1 ` ?

Reply:
Good question 😂